bindata/etcd/pod.yaml

```diff
@@ -193,15 +193,13 @@ ${COMPUTED_ENV_VARS}
         memory: 600Mi
         cpu: 300m
     readinessProbe:
-      httpGet:
-        port: 9980
-        path: readyz
-        scheme: HTTPS
-      initialDelaySeconds: 10
-      timeoutSeconds: 10
-      failureThreshold: 3
-      periodSeconds: 5
-      successThreshold: 1
+    tcpSocket:
```

Comment:
I posted this on slack already, but two general questions:
* why do we need a TCP probe when we have a working http probe? The zombies came from the forking of the etcdctl process, which doesn't exist anymore
* your indentation actually removes the probe configuration, I reckon that's also why the build is green :)

Reply:
I can try it with http. I had read a few places that tcp was best practices. Let me put back the working http, as I see what is going on with the indentation.